tests/test_device.py

```diff
@@ -86,7 +87,7 @@ def json(self):
     def raise_for_status(self):
         pass
 
-
+@pytest.mark.usefixtures("tape_mode")

```

Comment:
For the same reason as above, I would rather not have the test suite depend on any current tape mode mocking :)

Lets instead just run the tests in tape mode only.